using-timescaledb/continuous-aggregates.md

```diff
@@ -18,118 +18,154 @@ re-calculate all of the data on every refresh. Only new and/or
+hypertable. For example, we could create another continuous aggregate
+view that summarizes the hourly data.  updated hourly, you would
+write:
+
+```sql
+CREATE MATERIALIZED VIEW conditions_summary_hourly
 WITH (timescaledb.continuous) AS
 SELECT device,
-       time_bucket(INTERVAL '1 day', time) AS bucket,
+       time_bucket(INTERVAL '1 hour', time) AS bucket,
        AVG(temperature),
        MAX(temperature),
        MIN(temperature)
 FROM conditions
 GROUP BY device, bucket;
+
+-- Create the policy
+SELECT add_continuous_aggregate_policy('conditions_summary_hourly',
+	start_interval => INTERVAL '1 month',
+	end_interval => INTERVAL '1 h',
+	schedule_interval => INTERVAL '5 min');
 ```
 
+A `time_bucket` on the time partitioning column of the hypertable is
+required in all continuous aggregate views. If you do not provide one,
+you will get an error.
+
+When the view is created, it will (by default) be populated with data
+so that it contains an aggregate of the current data in
+`conditions`.
+
+It might, however, not always be desirable to populate the continuous
+aggregate. If the amount of data in `conditions` is large and new data
+is continuously being added, it is usually more meaningful to control
+the order in which the data is refreshed. For example, it might be
+more interesting to see the recent data but historical data can be
+deferred to later. In those cases, the `WITH NO DATA` option can be
+used to avoid aggregating the data during creation.
+
+You could then add a policy to control the refresh of the recent data,
+and also run manual refreshes using
+[`refresh_continuous_aggregate`][refresh_continuous_aggregate] to
+refresh the historical data in a controlled manner. For example, to
+refresh one month of data you could write:
+
+```sql
+CALL refresh_continuous_aggregate('conditions_summary_daily, '2020-05-01', '2020-06-01');
+```
+
+Unlike a regular materialized view, the refresh will only materialize
+the data within the window that has changed in the underlying
+hypertable since the last refresh. Therefore, if only a few buckets
+need updating, then the refresh is quick.
+
+Note that the end range is exclusive and aligned to the buckets of the
+continuous aggregate, so this will refresh only the buckets that are
+fully in the date range `['2020-05-01', '2020-06-01')`, that is, up to
+but not including `2020-06-01`. While it is possible to use `NULL` to
+indicate an open-ended range, we do not recommend doing so for tables
+that see continuous ingest of new data since that would trigger a
+refresh of buckets that are not yet filled completely.
+	
+You should avoid refreshing time intervals that still see a lot of
+writes, which is usually the last bucket of the continuous
+aggregate. These intervals are still changing and will not produce
+accurate aggregate anyway and refreshing unnecessarily will increase
+the write amplification, which will slow down the ingest rate of the
+hypertable.
+
+```sql
```

Comment:
If we believe there should at least suggest usage, maybe just describe in text without example? 

"If `NULL` arguments are supplied for the start or end interval, refreshing the continuous aggregate will start at the beginning or end, respectively, of the hypertable.  This might materialize a lot ...